project/src/main/java/com/google/autograder/servlets/CreateNewAssignmentServlet.java

```diff
@@ -44,7 +35,7 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
         int responseCode = 200;
 
         try {
-            byte[] postBodyData = buildPostBody(request).getBytes(StandardCharsets.UTF_8);
```

Comment:
This should be left as is, usually prefer to pass objects rather than primitive types (Strings, ints) so that you get type safety. Consider this contrived example:
```
class API {
  String UTF_8 = "UTF-8"; // Rather than "UTF_8"
}
```
then when you try to pass it to `getBytes` it'll fail. If you pass in a `Charset` then it's much less likely you'll pass in bad input. It's very common for this kind of mistake to happen when you're doing stuff with measurements (temperature, time, length, etc), which is why I think earlier I mentioned using time classes (Instant, Duration) rather than representing them as `long`

Reply:
+1